### DISCUSSION TITLE
Duplicates keep coming back

### DISCUSSION THREAD

**Discussion by lakemike** [immich-app/immich#11690] [General]
### The bug

Problem is that
- web version trashes duplicates, but the mobile version is unaware and does not manage the copy on the device
- when items are deleted from trash, the mobile version will re-upload them because there are now images in the device without cloud copies

### The OS that Immich Server is running on

QNAPLinux

### Version of Immich Server

v1.111.0

### Version of Immich Mobile App

v1.111.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
.
```

### Your .env content

```Shell
.
```

### Reproduction steps

```bash
1. web version puts duplicates into trash
2. trash gets deleted
3. mobile version reuploads duplicate images because duplicates have not been trashed on device
```

**Comment by bo0tzz** [comment]
This is a confluence of a few existing bugs/requests.

**Comment by C-Otto** [comment]
This is a duplicate of #8285
